src/modules/rlm_ldap/rlm_ldap.c

```diff
@@ -1911,6 +2109,11 @@ static int mod_bootstrap(module_inst_ctx_t const *mctx)
 	xlat = xlat_func_register_module(NULL, mctx, mctx->inst->name, ldap_xlat, FR_TYPE_STRING);
 	xlat_func_mono_set(xlat, ldap_xlat_arg);
 
+	snprintf(memberof_xlat, sizeof(memberof_xlat), "%s_memberof", mctx->inst->name);
+	xlat = xlat_func_register_module(NULL, mctx, memberof_xlat, ldap_memberof_xlat, FR_TYPE_BOOL);
```

Comment:
Can you change this so the function name is always created as `<inst>.<func name>`.  Maybe as a separate PR.  I'm guessing there's a bunch of xlat registrations which use the same pattern.